tests/tests/test_start_epoch_timestamp.rs

```diff
@@ -0,0 +1,76 @@
+#![allow(clippy::await_holding_refcell_ref)]
```

Comment:
This test can go in test_cluster_history.rs since it's testing the same instruction, and we can reuse the `create_copy_cluster_history_transaction` for the existing test there